src/OpenTelemetry.Exporter.OpenTelemetryProtocol/README.md

```diff
@@ -18,13 +18,17 @@ dotnet add package OpenTelemetry.Exporter.OpenTelemetryProtocol
 
 ## Configuration
 
-You can configure the `OtlpExporter` by following the directions below:
+You can configure the `OtlpExporter` through `OtlpExporterOptions` properties:
 
-* `ServiceName`: Name of the service reporting telemetry.
 * `Endpoint`: Target to which the exporter is going to send traces or metrics.
 * `Credentials`: Client-side channel credentials.
 * `Headers`: Optional headers for the connection.
 * `ChannelOptions`: gRPC channel options.
+* `ExportProcessorType`: Whether the exporter should use
+  [Batch or Simple base exporter](https://github.com/open-telemetry/opentelemetry-dotnet/tree/master/src/OpenTelemetry#introduction)
```

Comment:
pehaps link to the spec : https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/trace/sdk.md#built-in-span-processors

Reply:
Ah I linked to the API intro where it mentioned those but this is a better, more guided and specific reference. Thanks I'll update it.

Reply:
Most of the time, linking to spec is right, as its our source of truth! For anything specific to .NET implementation (this repo), we should link to this repo itself.